Sources/SourceKitLSP/Swift/CodeActions/ConvertIUOToProperOptional.swift

```diff
@@ -0,0 +1,59 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+import SwiftRefactor
+import SwiftSyntax
+
+/// Convert implicitly unwrapped optionals (IUOs) to proper optionals
+@_spi(Testing)
+public struct ConvertIUOToProperOptional: SyntaxRefactoringProvider {
+  public typealias Input = ImplicitlyUnwrappedOptionalTypeSyntax
+  public typealias Context = Void
+  public typealias Output = OptionalTypeSyntax
+
+  @_spi(Testing)
+  public static func refactor(syntax: Input, in context: Context) -> Output? {
```

Comment:
I think we can save all the type aliases if we spell out the types in the function signature. I think it also makes the code easier to read because you don’t always need to cross-reference the type aliases when reading the function signature of `refactor`.

```suggestion
  @_spi(Testing)
  public static func refactor(syntax: ImplicitlyUnwrappedOptionalTypeSyntax, in context: Void) -> OptionalTypeSyntax? {
```